driver-core/src/main/com/mongodb/client/model/vault/EncryptOptions.java

```diff
@@ -51,8 +51,9 @@ public EncryptOptions(final String algorithm) {
      *     <li>AEAD_AES_256_CBC_HMAC_SHA_512-Random</li>
      *     <li>Indexed</li>
      *     <li>Unindexed</li>
-     *     <li>RangePreview</li>
+     *     <li>Range</li>
      * </ul>
+     * Note: The Range algorithm is experimental only. It is subject to breaking changes.
```

Comment:
Is this still the case now its out of Preview?

Reply:
No, it should be marked as 'unstable.' I've removed the 'experimental' label and updated the status accordingly.